vtex/faststore
# feat: move status pages

## What's the purpose of this pull request?
This PR moves status pages (404, 505, offline) from gatsby-theme-store into the starters

## How to test it?
Make sure these pages remain the same after this PR
https://github.com/vtex-sites/btglobal.store/pull/679
https://github.com/vtex-sites/storecomponents.store/pull/1013
https://github.com/vtex-sites/marinbrasil.store/pull/545
https://github.com/vtex-sites/carrefourbrfood.store/pull/883
